test/unit/webgl/p5.RendererGL.js

```diff
@@ -743,6 +743,30 @@ suite('p5.RendererGL', function() {
       done();
     });
 
+    test('ambientLight() changes when metalness is applied', function (done) {
+      myp5.createCanvas(100, 100, myp5.WEBGL);
+      myp5.ambientLight(255, 255, 255);
+      myp5.noStroke();
+      myp5.metalness(100000);
+      myp5.sphere(50);
+      expect(myp5._renderer.mixedAmbientLight).to.not.have.members(
+        myp5._renderer.ambientLightColors);
+      done();
+    });
+
+    test('specularColor transforms to fill color when metalness is applied',
+      function (done) {
+        myp5.createCanvas(100, 100, myp5.WEBGL);
+        myp5.fill(0, 0, 0, 0);
+        myp5.specularMaterial(255, 255, 255, 255);
+        myp5.noStroke();
+        myp5.metalness(100000);
+        myp5.sphere(50);
+        expect(myp5._renderer.mixedSpecularColor).to.have.same.members(
```

Comment:
Thanks for adding the test! For this and the above test, I think order matters, so we probably want either `.to.have.same.ordered.members` (or maybe just `.to.deep.equal(...)`?)